<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>631</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>主界面</string>
    </property>
    <widget class="QTextBrowser" name="txt_arm">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>271</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>400</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;分割效果图&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>400</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;机械臂运行过程&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>20</width>
       <height>431</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>271</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>UIImage/White.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_display">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>展示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_armrun">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>182</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>系统设置</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>相机设置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_arm">
              <property name="text">
               <string>机械臂设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_network">
              <property name="text">
               <string>网络设置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="brn_users">
              <property name="text">
               <string>用户管理</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>移液类型</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="btn_plasma">
            <property name="text">
             <string>血浆分割</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btn_RBCs">
            <property name="text">
             <string>红细胞分割</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btn_Buffy">
            <property name="text">
             <string>白细胞和血小板分割</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>系统操作</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_run">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_stop">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_exit">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>相机设置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>机械臂设置</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>网络设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
